### PR TITLE
[AC-6224] Update 'Report Desired and Confirmed Mentors, and Goals' to SUI-based page

### PR DESCRIPTION
**Pull request by kitaka**
For details visit the related Accelerate PR: https://github.com/masschallenge/accelerate/pull/2060

### CODE REVIEW

**Comment by pbx** [comment]
Taking the code-quality task.

**Comment by kaymation** [comment]
qagley

File: accelerator_abstract/models/base_startup_mentor_tracking_record.py [hunk 1]
```diff
@@ -18,11 +18,8 @@
 """
 
 OTHER_MENTOR_HELP = """
-Please list Mentors you are already working with or would like to work with
-who are not currently listed in the MassChallenge Mentor Directory.
-You can list names, email addresses, company names, anything
-that would help identify them. We very much welcome Mentors who are not
-currently listed in the Mentor Directory.
+Add any mentors who you meet with and not listed in the MassChallenge
```
Comment (pbx) [inline accelerator_abstract/models/base_startup_mentor_tracking_record.py]: (Wording fix needed. See above.)

File: accelerator/migrations/0036_change_help_texts_for_notes_and_other_mentors_fields.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# Generated by Django 1.11.18 on 2019-02-14 13:16
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+PROGRAM_GOALS_HELP = "Submit the three goals you plan to "\
+                     "work on with your mentors during the "\
+                     "accelerator program."
+
+ADDITIONAL_MENTORS_HELP = "\nAdd any mentors who you meet with " \
```
Comment (pbx) [inline accelerator/migrations/0036_change_help_texts_for_notes_and_other_mentors_fields.py]: Needs a minor edit. "... mentors you meet with **who are** not listed ..."

File: accelerator_abstract/models/base_startup_mentor_tracking_record.py [hunk 3]
```diff
@@ -18,11 +18,8 @@
 """
 
 OTHER_MENTOR_HELP = """
```
Comment (pbx) [inline accelerator_abstract/models/base_startup_mentor_tracking_record.py]: This is called `ADDITIONAL_MENTORS_HELP` above; they should be the same.

File: accelerator_abstract/models/base_startup_mentor_tracking_record.py [hunk 4]
```diff
@@ -37,14 +34,14 @@ class BaseStartupMentorTrackingRecord(AcceleratorModel):
...
-        verbose_name="Other Mentors",
+        verbose_name="Additional Mentors",
         help_text=OTHER_MENTOR_HELP,
         blank=True,
         null=True)
     notes = models.TextField(
         verbose_name="Program Goals",
-        help_text="Submit the 3 goals you plan to work on with your Mentors "
-                  "during the Accelerator Program and report them here.",
+        help_text="Submit the three goals you plan to work on with your "
```
Comment (pbx) [inline accelerator_abstract/models/base_startup_mentor_tracking_record.py]: Seems like `PROGRAM_GOALS_HELP` should be defined above and used here.

File: accelerator_abstract/models/base_startup_mentor_tracking_record.py (hunk 1 above)
Comment (kitaka) [reply accelerator_abstract/models/base_startup_mentor_tracking_record.py]: @pbx What exactly do you mean by wording fix?

Comment (pbx) [reply accelerator_abstract/models/base_startup_mentor_tracking_record.py]: Sorry @kitaka I said "above" when it was actually below!  I was looking for a clarifying edit like "... mentors you meet with **who are** not listed ..."

Comment (kitaka) [reply accelerator_abstract/models/base_startup_mentor_tracking_record.py]: @hfreund your feedback on the above will be appreciated, just to keep the mockup in sync with the code.

Comment (hfreund) [reply accelerator_abstract/models/base_startup_mentor_tracking_record.py]: Paul's wording change is fine.

**Comment by pbx** [comment]
@kitaka - wording change that was made in [a0f806a](https://github.com/masschallenge/django-accelerator/pull/139/commits/a0f806afd8c029e410d7650c55475386beff9c1d) still needs to be made in the migration. Once that's done, I approve!

**Pull request merged**